Frontend/src/lib/pages/UsersPage.svelte

```diff
@@ -27,6 +29,13 @@
     
 </script>
 
+<Modal bind:showModal>
+    <h2 slot="header">
```

Comment:
Easier way to do this would be to make the title a prop and pass it in instead of making a slot for it. Unless you expect to do like \<a\> tags or \<i\> tags in the title or something

Reply:
Yeah I probably will use a prop here. I plan on using slots for the forms in some fashion. The exact implementation is not set though.